src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -337,6 +338,11 @@ protected virtual void PreShutdownCore(bool shutdownHost)
         {
         }
 
+        public int EvaluationPeriod
+        {
+            get { return pulseMaker == null ? 0 : pulseMaker.TimerPeriod; }
+        }
```

Comment:
Again, this should definitely not be in `DynamoModel` when multiple workspaces are impending.  We need to write code that anticipates that.